CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -152,23 +152,70 @@ internal static void AddDomainsOfExpertise(EngineeringModelSetup engineeringMode
         }
 
         /// <summary>
-        /// Adds the <see cref="Person"/> of the <see cref="Participant"/> of the <see cref="engineeringModelSetup"/>
-        /// to the target <see cref="HashSet{Person}"/> if this does not already contain them
+        /// Add <see cref="Person"/> and related objects: <see cref="PersonRole"/>, <see cref="ParticipantRole"/>, <see cref="Organization"/> to a specific hash set passed by reference
         /// </summary>
         /// <param name="engineeringModelSetup">
         /// The <see cref="EngineeringModelSetup"/> that contains the active <see cref="DomainOfExpertise"/> instances.
         /// </param>
         /// <param name="persons">
         /// The target <see cref="HashSet{Person}"/>
         /// </param>
-        internal static void AddPersons(EngineeringModelSetup engineeringModelSetup, ref HashSet<Person> persons)
+        /// <param name="personRoles">
+        /// The target <see cref="HashSet{PersonRole}"/>
+        /// </param>
+        /// <param name="participantRoles">
+        /// The target <see cref="HashSet{ParticipantRole}"/>
+        /// </param>
+        /// <param name="organizations">
+        /// The target <see cref="HashSet{Organization}"/>
+        /// </param>///
+        internal static void AddPersons(EngineeringModelSetup engineeringModelSetup, ref HashSet<Person> persons, ref HashSet<PersonRole> personRoles, ref HashSet<ParticipantRole> participantRoles, ref HashSet<Organization> organizations)
```

Comment:
Format parameters as one per new line (including first one), since this is too long.

Reply:
```
internal static void AddPersons(EngineeringModelSetup engineeringModelSetup, ref HashSet<Person> persons, ref HashSet<PersonRole> personRoles, ref HashSet<ParticipantRole> participantRoles, ref HashSet<Organization> organizations)
```
vs
```
internal static void AddPersons(
    EngineeringModelSetup engineeringModelSetup,
    ref HashSet<Person> persons,
    ref HashSet<PersonRole> personRoles,
    ref HashSet<ParticipantRole> participantRoles,
    ref HashSet<Organization> organizations)
```

Reply:
Ok